viz.yaml

```diff
@@ -41,5 +44,12 @@ process:
     depends: 
       waterUse: waterUse-data
 visualize:
+  -
+    id: wateruse-treemap
+    location: cache/treemaps/
+    visualizer: treemap
+    scripts: scripts/visualize/treemap.R
+    depends:
+      waterUse_clean: calc-waterData
```

Comment:
So we can call:
```
viz[['depends']][['waterUse_clean']]
```
instead of 
```
viz[['depends']][1]
```

This is more important when there are a bunch of depends and you are trying desperately to remember if it's the first, or second, or whatever dependency.

The discussion is here:
https://github.com/USGS-VIZLAB/vizlab/pull/112
and
https://github.com/USGS-VIZLAB/vizlab/issues/94